src/deepsparse/loggers/prometheus_logger.py

```diff
@@ -90,18 +91,19 @@ def log(self, identifier: str, value: Any, category: MetricCategories):
         :param category: The metric category that the log belongs to
         """
 
-        # needs to adhere to
-        # https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels
-        formatted_identifier = (
-            identifier.replace(".", "__").replace("-", "__").replace("/", "__")
-        )
-        prometheus_metric = self._prometheus_metrics.get(formatted_identifier)
-        if prometheus_metric is None:
-            prometheus_metric = self._add_metric_to_registry(
-                formatted_identifier, category
+        for identifier, value in unwrap_logs_dictionary(value, identifier):
+            # needs to adhere to
+            # https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels
+            formatted_identifier = (
+                identifier.replace(".", "__").replace("-", "__").replace("/", "__")
             )
-        prometheus_metric.observe(self._validate(value))
-        self._export_metrics_to_textfile()
+            prometheus_metric = self._prometheus_metrics.get(formatted_identifier)
+            if prometheus_metric is None:
+                prometheus_metric = self._add_metric_to_registry(
+                    formatted_identifier, category
+                )
+            prometheus_metric.observe(self._validate(value))
+            self._export_metrics_to_textfile()
```

Comment:
might want to break this section out into multiple helper functions, especially as we are getting into using more prometheus functionality

Reply:
I am doing it atm while working on the POC for matching the logging targets to appropriate prometheus metrics. Let's turn the blind eye in the context of this PR.